tests/nlu/base/test_utils.py

```diff
@@ -237,3 +243,10 @@ def test_custom_token_name():
     actual = EndpointConfig.from_dict(test_data)
 
     assert actual.token_name == "test_token"
+
+
+def test_validate_pipeline_yaml():
+    validate_pipeline_yaml(CONFIG_DEFAULTS_PATH)
+
+    with pytest.raises(InvalidConfigError):
+        validate_pipeline_yaml("data/test_config/example_config.yaml")
```

Comment:
could we add tests for a fully specified pipeline and a pipeline which uses a template?